catalog/transformer/catalog-transformer-html/src/main/java/ddf/catalog/transformer/html/HtmlMetacardTransformer.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.html;
+
+import ddf.catalog.data.BinaryContent;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.BinaryContentImpl;
+import ddf.catalog.transform.CatalogTransformerException;
+import ddf.catalog.transform.MetacardTransformer;
+import ddf.catalog.transformer.html.models.HtmlExportCategory;
+import ddf.catalog.transformer.html.models.HtmlMetacardModel;
+import java.io.ByteArrayInputStream;
+import java.io.Serializable;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.lang.StringUtils;
+
+public class HtmlMetacardTransformer implements MetacardTransformer {
+
+  private HtmlMetacardUtility htmlMetacardUtility;
```

Comment:
✏️ I don't see much value in separating out the class that does the majority of the work into a utility class. It doesn't appear to be used by any other consumer but this transformer so there's no value in refactoring it; putting its functionality in here would improve isolation and, I think, improve clarity.

Reply:
@jrnorth The utility class is being used by both the `HtmlMetacardTransformer` and `HtmlQueryResponseTransformer` which is why I broke the functionality out into a separate class. 

Also following the refactor using polymorphism for the different value models each value model also uses it to render a value model template.